core/tx_pool.go

```diff
@@ -68,8 +73,10 @@ type TxPool struct {
 	events       event.Subscription
 	localTx      *txSet
 	mu           sync.RWMutex
-	pending      map[common.Hash]*types.Transaction // processable transactions
-	queue        map[common.Address]map[common.Hash]*types.Transaction
+
+	pending map[common.Address]TxList          // All currently processable transactions
```

Comment:
Pre-patch this is a map of addresses to a single transaction. Shouldn't that still be the case, since only one transaction per account is processable at any one time?